docs/src/cli-reference/tensile_create_library_cli.rst

```diff
@@ -0,0 +1,154 @@
+.. meta::
+  :description: Tensile documentation and API reference
+  :keywords: Tensile, GEMM, Tensor, ROCm, API, Documentation
+
+.. _tensilecreatelibrary-cli-reference:
+
+====================
+TensileCreateLibrary
+====================
+
+``TensileCreateLibrary`` is a command-line tool that generates libraries and code object files for a set of supplied logic files.
+
+Syntax
+------
+
+.. code-block::
+
+    TensileCreateLibrary [OPTIONS...] <LOGIC DIRECTORY> <OUTPUT DIRECTORY> <RUNTIME LANGUAGE>
+
+Required arguments
+-------------------
+
+Here is the list of required arguments for invoking the ``TensileCreateLibrary`` command.
+
+.. list-table:: Required arguments
+  :header-rows: 1
+
+  * - Argument
+    - Description
+
+  * - \<LOGIC DIRECTORY\>
+    - Absolute path for logic files. The logic files are generally found in either of the following locations:
+      |br| \- The build directory under ``3_LibraryLogic``, when generated by the Tensile program. |br|
+      |br| \- In a project that hosts pregenerated logic files, such as `rocBLAS <https://github.com/ROCm/rocBLAS/tree/develop/library/src/blas3/Tensile/Logic>`_. |br|
+
+  * - \<OUTPUT DIRECTORY\>
+    - Absolute or relative path to the output directory where build artifacts are placed.
+
+  * - \<RUNTIME LANGUAGE\>
+    - Runtime language out of OCL, HIP, or HSA
+
+Options
+-------
+
+Here is the list of optional arguments for invoking the ``TensileCreateLibrary`` command.
+
+.. list-table:: Options
+  :header-rows: 1
+
+  * - Option
+    - Description
+
+  * - \-\-architecture=ARCHITECTURE
+    - Architectures to generate a library for. When specifying multiple options, use quoted and semicolon-delimited
+      architectures such as \-\-architecture='gfx908;gfx1012'.
+      Supported architectures include: all; gfx000; gfx803; gfx900; gfx900:xnack-; gfx906; gfx906:xnack+; gfx906:xnack-; gfx908; gfx908:xnack+;
+      gfx908:xnack-; gfx90a; gfx90a:xnack+; gfx90a:xnack-; gfx940; gfx940:xnack+; gfx940:xnack-; gfx941; gfx941:xnack+;
+      gfx941:xnack-; gfx942; gfx942:xnack+; gfx942:xnack-; gfx1010; gfx1011; gfx1012; gfx1030; gfx1031; gfx1032; gfx1034; gfx1035;
+      gfx1100; gfx1101; gfx1102.
```

Comment:
@bstefanuk Are the supported architectures the same as the supported architectures for ROCm? If yes, then we don't need to mention these.

Reply:
No they're not, Tensile supports a sub-set of available cards, and it often takes some time for us to bring support for new architectures, so this is important information.